doc/contact.rst

```diff
@@ -16,6 +16,9 @@ If you want to talk with us you may:
 - join chats at **#autocrypt on freenode** or through `matrix mirror
   <https://riot.im/app/#/room/#autocrypt:matrix.org>`_.
 
+- if you want to more privately talk to project members please mail
```

Comment:
This sounds odd.  if someone wants to talk privately to members of the project, and they know which member they want to talk to, they can contact that member directly (it's easy to find our contact information by reading the mailing list archives).

So we're trying to handle a situation where they don't know which project member they want to talk to specifically, and they can't or don't want to use the public list.  right?

Let's try to walk through the specific reasons that they want to talk privately.  depending on those reasons, they may have different needs.

 * security concerns -- these folks might prefer encrypted e-mail.  Can we give it to them?
 * difficulty subscribing to the mailing list -- this should be addressed from the mailing list itself, not here
 * technical questions, but discomfort with discussing them publicly -- we're saying below that this is an inappropriate use of this address
 * media inquiries -- perhaps this deserves its own rationale, or its own e-mail address?
 * any other reason?

Reply:
that would be "press@"